src/plugins/discover/public/components/discover_grid/discover_grid.scss

```diff
@@ -65,14 +65,6 @@
   min-height: 0;
 }
 
-.dscDiscoverGrid__footer {
-  flex-shrink: 0;
-  background-color: $euiColorLightShade;
-  padding: $euiSize / 2 $euiSize;
-  margin-top: $euiSize / 4;
```

Comment:
Here you can use more specific EUI sizing variables, for example `$euiSizeS`, `$euiSizeXS` that way you don't need the divisions.

Reply:
@andreadelrio This code is removed now. In `src/plugins/discover/public/components/discover_grid/discover_grid_footer.tsx` I use emotion instead.